src/time-range-picker/combobox-picker/combobox-picker.tsx

```diff
@@ -248,9 +241,8 @@ export const ComboboxPicker = ({
         if (
             nodeRef.current &&
             !nodeRef.current.contains(event.relatedTarget) &&
-            !dropdownOpen
+            !dropdownOpen // Necessary because dropdown floating ui is not a child
```

Comment:
`onBlur` fires twice with shift + tab due to overlap with `handleClose`

suggest to:
1. check `activeTimeSelector` is truthy here (similar to the focus check in the `Select` component)
2. trigger `onBlur` in `handleClose`